src/bin/menu_bar_system.rs

```diff
@@ -9,7 +9,7 @@ extern crate gtk;
 use gio::prelude::*;
 use gtk::{
     AboutDialog, AboutDialogExt, BoxExt, ContainerExt, DialogExt, GtkApplicationExt, GtkWindowExt,
-    Inhibit, LabelExt, SwitchExt, ToVariant, WidgetExt,
+    LabelExt, SwitchExt, ToVariant, WidgetExt,
```

Comment:
Why don't you just import the gtk prelude here (and in other places)?

Reply:
Mostly because it was already imported this way :wink:
Want me change this too ?

Reply:
Yes please :)